pkg/sdk/testint/client_unsafe_extensions_integration_test.go

```diff
@@ -13,13 +13,13 @@ func TestInt_Client_UnsafeQuery(t *testing.T) {
 	ctx := testContext(t)
 
 	t.Run("test show databases", func(t *testing.T) {
-		sql := fmt.Sprintf("SHOW DATABASES LIKE '%%%s%%'", testDb(t).Name)
+		sql := fmt.Sprintf("SHOW DATABASES LIKE '%%%s%%'", testDb(t).Name.Name())
 		results, err := client.QueryUnsafe(ctx, sql)
 		require.NoError(t, err)
 
 		assert.Len(t, results, 1)
 		row := results[0]
-		assert.Equal(t, testDb(t).Name, *row["name"])
+		assert.Equal(t, testDb(t).Name.Name(), *row["name"])
```

Comment:
Maybe instead of having `Name.Name()` which is ugly we should have ID().Name() ? (in all other places too)

Reply:
Mostly reverted, but when left I used `ID()`